araig_test_gui/CMakeLists.txt

```diff
@@ -0,0 +1,206 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(araig_test_gui)
+
+## Compile as C++11, supported in ROS Kinetic and newer
+# add_compile_options(-std=c++11)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS
+  roscpp
```

Comment:
As far as I can see, there are no CPP files here, you should not need this dep. PLease remove it if that is the case.